coda/coda_mdstore/models.py

```diff
@@ -4,29 +4,32 @@
 class Bag(models.Model):
     name = models.CharField(
         max_length=255,
-        help_text="Name of Bag",
-        primary_key=True
-    )
+        primary_key=True,
+        help_text="Name of Bag")
```

Comment:
Not to be picky or anything, but if you're already making purely stylistic changes, something to consider would be changing all the quotes here to single quotes.